src/main/kotlin/org/pkl/lsp/ast/Terminal.kt

```diff
@@ -151,9 +151,9 @@ fun TreeSitterNode.toTerminal(parent: PklNode): Terminal? {
       "identifier" -> TokenType.Identifier
       "docComment" -> TokenType.DocComment
       "escapeSequence" -> TokenType.CharacterEscape
+      "\n" -> TokenType.Whitespace
```

Comment:
This now gets emitted as a token because `\n` is explicitly provided as an alternative for `propertyCallExpr`.